dev/wandb_logger.py

```diff
@@ -0,0 +1,48 @@
+import os
+import time
+import argparse
+import wandb
+
+# Please follow https://docs.wandb.ai/quickstart and setup wandb server and python client
+
+logging_metrics = {
+    'eval'      : 'Hellaswag Evaluation',
+    'tel'       : 'Validation Loss',
+    'trl'       : 'Training Loss',
+    'norm'      : 'Normalized Gradient',
+    'lr'        : 'Learning Rate',
+    'latency'   : 'Latency Per Step (ms)',    
+    'mfu'       : 'A100 FP16 MFU %',
+    'tokens'    : 'Tokens Per Second'
+}
+
+def parse_line(line):
+    for metric, header in logging_metrics.items():
+        if metric in line:
+            parts = line.split()
+            step = int(parts[0].split(":")[1])
+            value = float(parts[1].split(":")[1])
+            wandb.log({header: value}, step=step)
```

Comment:
`parse_line` suggest we're parsing line, but then we do more than that and this is the actual logging line to wandb. either rename the function or parse_line returns a dict, and then call wandb on it outside or so.